src/ui/public/persisted_log/__tests__/persisted_log.js

```diff
@@ -15,27 +11,16 @@ const payload = [
   { first: 'bruce', last: 'wayne' }
 ];
 
-
-function init() {
-  ngMock.module('kibana/persisted_log', function ($provide) {
-    // mock storage service
-    $provide.service('localStorage', function () {
-      this.get = sinon.stub();
-      this.set = sinon.stub();
-      this.remove = sinon.spy();
-      this.clear = sinon.spy();
-    });
-  });
-
-  ngMock.inject(function ($injector) {
-    storage = $injector.get('localStorage');
-    PersistedLog = $injector.get('PersistedLog');
-  });
-}
-
 describe('PersistedLog', function () {
```

Comment:
Perhaps outside the scope of this PR, but this test can be converted to Jest by renaming it `persisted_log.test.js` and moving it out of this directory into the same directory as `persisted_log.js`. And then all the `it` calls can be renamed to be `test`, which is purely cosmetic.